Document/0x06d-Testing-Data-Storage.md

```diff
@@ -715,13 +715,54 @@ The following is [sample Swift code for excluding a file from a backup](https://
 
 #### Dynamic Analysis
 
-After the app data has been backed up, review the data that's in the backed up files and folders. The following directories should be reviewed for sensitive data:
+In order to test the backup, you obviously need to create one first. The most common way to create a backup of an iOS device is by using iTunes, which is available for Windows, Linux and of course macOS. When creating a backup via iTunes you can always only backup the whole device and not select just a single app. Make sure that the option "Encrypt local backup" in iTunes is not set, so that the backup is stored in cleartext on your hard drive.
 
-- Documents/
-- Library/Application Support/
-- Library/Preferences/
+After the iOS device has been backed up through iTunes you need to retrieve the file path of the backup, which are different locations on each OS. The official Apple documentation will help you to [locate backups of your iPhone, iPad, and iPod touch](https://support.apple.com/en-us/HT204215 "Locate backups of your iPhone, iPad, and iPod touch").
+
+When you want to navigate to the iTunes backup folder on macOS Mojave and later you will get the following error (even as root):
+
+```bash
+$ pwd
+/Users/foo/Library/Application Support
+$ ls -alh MobileSync
+ls: MobileSync: Operation not permitted
+```
+
+This is not a permission issue of the backup folder, but a new feature in macOS Mojave. Allow the Terminal or iTerm or whatever your terminal software is full disk access, which is explained [here](http://osxdaily.com/2018/10/09/fix-operation-not-permitted-terminal-error-macos/ "Fix Terminal “Operation not permitted” Error in MacOS Mojave") in detail.
```

Comment:
Do we want to provide guidance for older mac os x versions as well?

Reply:
I just added one more sentence to clarify that till High Sierra this is not an issue.

Reply:
Also added a link from Jeroen Beckers about how to analyse when having only encrypted backups